cli/cmd/sync.go

```diff
@@ -124,6 +125,10 @@ func sync(cmd *cobra.Command, args []string) error {
 	// in a cloud sync environment, we pass only the relevant environment variables to the plugin
 	osEnviron := os.Environ()
 
+	analytics.TrackSyncStarted(ctx, invocationUUID, analytics.SyncStartedEvent{
```

Comment:
Easy to start started for all sources + destinations in a single place, for sync finished see below